lib/connection/process.coffee

```diff
@@ -120,6 +120,9 @@ module.exports = jlprocess =
       server.close()
       fn()
 
+  onStdout: (f) -> @emitter.on 'stdout', f
+  onStderr: (f) -> @emitter.on 'stderr', f
```

Comment:
In principle it'd be right to dispose of the listeners that are created [here](https://github.com/JunoLab/atom-julia-client/blob/omm/modules/lib/runtime/console.coffee#L32-L33). In practice it shouldn't matter, because when the package is deactivated the whole `Emitter` will be thrown away at once.